checkov/cloudformation/checks/resource/aws/AmazonMQBrokerPublicAccess.py

```diff
@@ -0,0 +1,28 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.cloudformation.checks.resource.base_resource_check import BaseResourceCheck
+
+class AmazonMQBrokerPublicAccess(BaseResourceCheck):
+
+    def __init__(self):
+        name = "Ensure Amazon MQ Broker should not have public access"
+        id = "CKV_AWS_166"
```

Comment:
Can you change the id to `CKV_AWS_69` then it correspond to the Terraform check

Reply:
@gruebel changed the id to **CKV_AWS_69**